cli/src/commands/init/index.ts

```diff
@@ -188,33 +220,41 @@ export default class Init extends BaseCommand<typeof Init> {
 
   async getPackageManager() {
     const packageManager = await this.guessPackageManager();
```

Comment:
Should we use `flags.packageManager` here as a default?

Reply:
fixed with https://github.com/xataio/client-ts/pull/1021/commits/2f010952c16852653f6fd00f27c7928587d5e826